README.md

```diff
@@ -43,6 +43,15 @@ To run e2e tests, you must have Java installed. [[download Java here](https://ja
 
 
 ## Deployment
-The source files for the beta deployment are contained in the `./docs/` dir, and are live. Any changes to this dir's contents will be automatically deployed, so please take care when making modifications to this location. 
+The source files for the beta deployment are contained in the `./docs/` dir, and are live. Any changes to this dir's contents will be automatically deployed, so please take care when making modifications to this location.
 
 To update the dist bundle, run ```$ npm run build```, and copy the generated files from `./dist/` to `./docs/`, taking care to not delete the CNAME file in `./docs/`.
+
+Add your Google Analyitics id in `src/main.js`:
```

Comment:
Please remove this section, I would like the analytics to be disabled entirely when in a dev environment by default to avoid sending events that are not from the deployed chooser. 

Bonus points if you can find a way to enable the event sending functionality explicitly with a flag on `npm run start` or `npm run dev` to allow for testing of analytics code when needed.